docs/publish/publish.md

```diff
@@ -20,7 +20,7 @@ The following table summarizes the primary publication methods that can be used
 |:---------|:------------|
 |[Sideloading](../testing/test-debug-office-add-ins.md#sideload-an-office-add-in-for-testing)|As part of your development process, to test your add-in running on Windows, iPad, Mac, or in a browser. (Not for production add-ins.) |
 |[AppSource](#appsource)|To distribute your add-in publicly to users.|
-|[Microsoft 365 admin center](/microsoft-365/admin/manage/test-and-deploy-microsoft-365-apps)|In a cloud deployment, to distribute your add-in to users in your organization by using the Microsoft 365 admin center. This is done through [Integrated Apps](/microsoft-365/admin/manage/test-and-deploy-microsoft-365-apps) or [Centralized Deployment](/microsoft-365/admin/manage/centralized-deployment-of-add-ins). |
+|[Integrated apps portal](/microsoft-365/admin/manage/test-and-deploy-microsoft-365-apps) in the [Microsoft 365 admin center](/microsoft-365/admin/manage/test-and-deploy-microsoft-365-apps)|To distribute your add-in to users in your organization.|
```

Comment:
Both of these link to the same place. Can we keep only one of them hyperlinked or change one of them to a different (and appropriate) destination?